app/controllers/chugtypes_controller.rb

```diff
@@ -9,9 +9,11 @@ def index
 
   def show
     @pagy, @chugs_all_newest = pagy(@chugtype.chugs.order('created_at DESC'), items: 10, page: params[:page])
-    @chugs_unique = Chug.all.order('time ASC, created_at ASC')
+    @chugs_unique = (Chug.all.order('time ASC, created_at ASC')
```

Comment:
Deze query word zo wel heel groot en onoverzichtelijk, ik zou een scope gaan definiëren op het model zodat je Chug.external of Chug.unique kan gebruiken. 

https://guides.rubyonrails.org/active_record_querying.html#scopes